packages/next/src/client/components/nav-failure-handler.ts

```diff
@@ -3,6 +3,7 @@ import { createHrefFromUrl } from './router-reducer/create-href-from-url'
 
 export function handleHardNavError(error: unknown): boolean {
   if (
+    error &&
```

Comment:
Seems redundant now that we also check it before we call `handleHardNavError`. Makes more sense to handle it at the callsite given this function is about handling an error. Wouldn't make sense to pass something in here that's not an error

```suggestion
```

Reply:
Left here as well in case we get a bad error from uncaught exception/rejection as well

Reply:
I missed that we also call it in https://github.com/vercel/next.js/blob/a10f0e93f3e580604fddb0ec1f13bcd53c8b1b0a/packages/next/src/client/components/nav-failure-handler.ts#L33